examples/vrf/Cargo.toml

```diff
@@ -13,18 +13,18 @@ documentation = "https://docs.rs/commonware-vrf"
 [dependencies]
 commonware-cryptography = { workspace = true }
 commonware-macros = { workspace = true }
-commonware-p2p = { workspace = true } 
+commonware-p2p = { workspace = true }
 commonware-runtime = { workspace = true }
 commonware-utils = { workspace = true }
-bytes = { workspace = true } 
+bytes = { workspace = true }
 rand = { workspace = true }
-tracing = { workspace = true } 
-prost = { workspace = true } 
+tracing = { workspace = true }
+prost = { workspace = true }
 futures = { workspace = true }
 clap = { workspace = true }
 tracing-subscriber = { workspace = true }
 prometheus-client = { workspace = true }
 governor = { workspace = true }
 
 [build-dependencies]
-prost-build = { workspace = true } 
\ No newline at end of file
+prost-build = { workspace = true }
```

Comment:
Set `"files.insertFinalNewline": true` in VSCode

Reply:
will do that going forward. Thus far, I haven't used it and figure we keep it that way on this one.